x-pack/plugins/security/public/account_management/api_keys_page/api_keys_empty_prompt.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { EuiLink, EuiEmptyPrompt } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
+
+export interface ApiKeysEmptyPromptProps {
+  error?: Error;
+}
+
+export const ApiKeysEmptyPrompt: FunctionComponent<ApiKeysEmptyPromptProps> = ({
+  error,
+  children,
+}) => {
+  const { services } = useKibana();
+
+  if (error) {
+    const { statusCode, message = '' } = (error as any).body ?? {};
+
+    if (statusCode === 400 && message.indexOf('[feature_not_enabled_exception]') !== -1) {
```

Comment:
Let's chat quick about this logic tomorrow. I have a couple of ideas that might simplify/standardize things a bit

Reply:
So I've just noticed that the class you recommended me using is server-side code: `x-pack/plugins/security/server/authentication/api_keys` so not sure it's expected to be used on the client. 

The class also requires a logger and cluster client instance, non of which I have access to in the browser so I will  keep this simple for now and use a functional approach directly in the component source.